docs/inspectAnnotation.md

```diff
@@ -0,0 +1,28 @@
+# Inspect Annotation
+
+Baremetalhost's(BMH) _Status_ sub-resource contain a _hardware_ key
+which contains the result of introspection which is carried out during
+BMH registration.
+
+In some circumstances it may be desirable to disable this inspection process,
+and provide data from external source.  The _Inspect Annotation_ provides some
+interfaces to enabled this.
```

Comment:
nit: we should say in here that the structure of the annotation's value should match the `hardware` field's schema. We can do that as an update if there are no other reasons to change this PR.

Reply:
Ack, I'm happy to add it here - we should perhaps also mention it in the status annotation doc, and that could link to this mentioning that it's an alternative API if you only want to update the hardware field of the status?

Reply:
I should also perhaps mention that the hardware details can be a subset of the full hardware field schema since https://github.com/metal3-io/baremetal-operator/pull/803

Reply:
Ok updated in the latest revision